src/parser/pdb-parser.ts

```diff
@@ -301,7 +301,9 @@ class PdbParser extends StructureParser {
 
             if (!isLegacy) {
               if (isPdbqt) {
-                element = line.substr(12, 2).trim()
+                element = line.substr(76, 3).trim()
```

Comment:
I'm going to take it that pdbqt is never missing this field? (i.e. we don't need to fallback to `line.substr(12,2)`?

Reply:
You are correct, that's a given, otherwise the programs that need this format can't work properly